.gitignore

```diff
@@ -14,3 +14,4 @@ npm-debug.log
 *.log
 
 /.nyc_output/
+test/fixtures/*/dist/**
```

Comment:
This needs updating for the new fixture location, and the committed `dist` files from the last commit to be deleted.

This could also be combined with the repo-root `/dist` entry on line 10 in this file? :-)